packages/dds/tree/src/feature-libraries/modular-schema/modularChangeTypes.ts

```diff
@@ -60,6 +60,10 @@ export interface ModularChangeset extends HasFieldChanges {
 	 * If this count is greater than 0, it will prevent the changeset from being applied.
 	 */
 	readonly constraintViolationCount?: number;
+	/** Global no change constraint that gets violated whenever the changeset is rebased */
+	readonly noChangeConstraint?: NoChangeConstraint;
+	/** Global no change constraint for reverts that gets violated whenever the reverted is rebased */
```

Comment:
I've been struggling to find a clear way to describe this, how about this?
```suggestion
	/** Global {@link noChangeConstraint | no change constraint} to impose on reverts of the changeset */
```